test/templates/java-spring/no-params/README.md

```diff
@@ -0,0 +1,104 @@
+<h1 align="center">Java Spring generator</h1>
+<p align="center">
+  <em>Use your AsyncAPI definition to generate java code to subscribe and publish messages</em>
+</p>
+<br><br>
+
+## Usage
+
+### AsyncAPI extensions
+In order for the generator to know what names to use for some methods it's necessary to make use of [AsyncAPI specification extensions](https://www.asyncapi.com/v1/spec.html#Specification-Extensions). The extension to use is **x-service-name** and must be applied to all topics.
+
+here is an example of how to use it:
```

Comment:
- Capitalization
- Precision
"Here is an example for a single topic:"

Reply:
The suggestions have been brought on to the java spring 2.0